### DISCUSSION TITLE
Does VisiData have an official Mastodon address?

### DISCUSSION THREAD

**Discussion by frosencrantz** [saulpw/visidata#1614] [General]
The birdsite seems to be in turmoil.  Likely it will eventually be fine, but it seems like there are a lot of people who like VisiData that are also on Mastodon.  It seems like it would be good to make sure there is an official account somewhere  on Mastodon.

**Comment by aborruso** [comment]
> a lot of people who like VisiData that are also on Mastodon.

What's you account? Mine is this `@aborruso@mastodon.uno`

**Comment by anjakefala** [comment]
VisiData does not currently have a Mastodon that I recall!

I think if it does get one, someone else would have to drive that. I am taking advantage of the twitter collapse to break up with social media.

Does anyone have Mastodon servers that they think would make a good home for VisiData?

**Comment by frosencrantz** [comment]
I’m not on social media.  But I do like to keep on what people are saying.  I don’t have 

- The Python Bytes podcast team is using https://fosstodon.org/.   This might be a good choice because it is for open source
- Will McGugan has gone with https://mastodon.social/ at least for his account.
- And Simon Willison created his own host: http://simonwillison.net/ 


But there are many choices.  Simon has created a git-scraping repository  listing all Mastodon instances, including ones open for new accounts.  It is readable with our favorite data exploring tool: https://raw.githubusercontent.com/simonw/scrape-instances-social/main/instances.json

**Comment by ajkerrigan** [comment]
Like @frosencrantz mentioned, Fosstodon seems like a pretty reasonable default. That said, opting out of a new platform altogether sounds pretty darn sensible. (I'm not a prolific social media user myself, due to some combination of choice and incompetence. I couldn't volunteer in good conscience to drive a project's Mastodon presence, so wouldn't expect someone else to either.)

**Comment by frosencrantz** [comment]
Things are getting worse on the birdsite, and it feels like it might be a good time to hedge bets and create a Mastodon account.  Simon Willison is saying he is getting very good engagement with less followers. I find that tracking activity on Mastodon is much easier than the birdsite, no account needed.  After seeing a few posts on birdsite you need to have an account.

In August I started informally tracking follower counts for the VisiData account, and noticed the highest count was on 2022/10/22  with 1798, it is now at 1771.  It did go down 10 by Nov 6th.  Before October I had only noticed increases.  So there are people who would be interested in VisiData leaving Twitter.  I noticed there was a little bit of calm mid-November, and there was a small increase to 1791.  I cannot track individual followers, so I cannot say whether these were recent  or long time followers.

It seems like having a social media presence is helpful for driving awareness and interest for VisiData.  So the question is what accounts to have. Like @ajkerrigan, I'm not volunteering to drive a Mastodon presence, so  it is very easy for me to volunteer others, particularly at a busy time of year.

The account would need to be created, and any links from source code or visidata.org would need to be added.  But after that I wonder if just cross posting between the sites is all that is needed?  I think the Python Bytes team has both but has a Mastodon first policy, which seems reasonable.  If Simon's experience is similar for others, it could be that people willing to go to Mastodon are early adopters, and it might be a good group to expose to VisiData.


And there is already some activity on VisiData:
   - https://mastodon.social/tags/visidata
   - https://mastodon.social/tags/HanukkahOfData
